networking/hardware_networks/about-sriov.adoc

```diff
@@ -31,9 +31,9 @@ It performs the following functions:
 The Operator provisions the following components:
 
 SR-IOV network configuration daemon::
-A DaemonSet that is deployed on worker nodes when the SR-IOV Operator starts. The daemon is responsible for discovering and initializing SR-IOV network devices in the cluster.
+A DaemonSet that is deployed on worker nodes when the SR-IOV Network Operator starts. The daemon is responsible for discovering and initializing SR-IOV network devices in the cluster.
```

Comment:
s/DaemonSet/ daemon set 

per updated guidance in https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references